ROMFS/px4fmu_common/init.d/rc.logging

```diff
@@ -24,4 +24,3 @@ fi
 logger start -b ${LOGGER_BUF} -t ${LOGGER_ARGS}
 
 unset LOGGER_ARGS
-unset LOGGER_BUF
```

Comment:
Did you mean to unset this in `rcS`?

Reply:
Hi @bkueng,

I did. I made that change because it is set twice in rcS and I'm trying to unset script variables at the same level they first get set. Ultimately, I think I can migrate the `set LOGGER_BUF` instances into rc.logging, at which point the unset would get moved back as well.

Given that goal, let me know if you'd prefer it be left as-is for the time being, and I'm happy to move it back.

-Mark

Reply:
Ah ok, I did not see it's already there. All good then for now, thanks!